server/live-loan-router.js

```diff
@@ -123,12 +123,12 @@ module.exports = function liveLoanRouter(cache) {
 	});
 
 	// Filter URL Router
-	router.use('/f/:id([a-zA-Z0-9%,_-]{0,})/url/:offset(\\d{0,})', async (req, res) => {
+	router.use('/f/:id([a-z.A-Z.0-9.%,_-]{0,})/url/:offset(\\d{0,})', async (req, res) => {
```

Comment:
On this line and on line 131, the `.` only needs to be included once, and it should be escaped: `[a-zA-Z0-9\.%,_-]{0,}`